target/snitch_cluster/sw/apps/snax-streamer-gemm/src/snax-streamer-gemm.c

```diff
@@ -36,13 +34,13 @@ int main() {
     // Wait for DMA to finish
     snrt_cluster_hw_barrier();
 
-    if (snrt_is_compute_core()) {
-        uint32_t gemm_start = snrt_mcycle();
+    if (snrt_global_core_idx() == 0) {
+        // uint32_t gemm_start = snrt_mcycle();
```

Comment:
Why does this need a change?

Reply:
Later we will add more compute cores and each of them will have an idx. Here we specify the core id to be consistent with multi-compute core situation.